iib/web/iib_static_types.py

```diff
@@ -162,7 +162,7 @@ class MergeIndexImagesPayload(TypedDict):
     build_tags: NotRequired[List[str]]
     deprecation_list: NotRequired[List[str]]
     distribution_scope: NotRequired[str]
-    graph_update_mode: NotRequired[str]
+    graph_update_mode: NotRequired[GRAPH_MODE_LITERAL]
```

Comment:
Is it possible to do this in the previous commit instead of this one?

Reply:
The other way around. The `graph_update_mode` should not be present in the first commit.